src/keycloak/chart/values.schema.json

```diff
@@ -453,8 +453,37 @@
             "type": "string"
           }
         },
-        "required": [ "name", "value" ]
+        "required": ["name", "value"]
+      }
+    },
+    "themeCustomizations": {
+      "type": "object",
+      "properties": {
+        "resources": {
+          "type": "object",
+          "properties": {
+            "images": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string"
+                  },
```

Comment:
Should this have an enum restriction on the four expected files? I know identity-config will only copy those four anyways, but having it in the values schema might surface it as an error rather than silently ignoring.

Reply:
Sure! Added!